### ISSUE TITLE
Feature: per-step accent / velocity

### ISSUE THREAD

**Issue by pierluigi** [pierlo-upitup/polyrhythmical#5]
Allow users to easily select a different velocity per step. A simple approach a-la Redrum (Reason drum machine) would be a good start!

**Comment by glitched01** [comment]
UI Ideas:

Velocity value is indicated by color intensity (or hue, saturation, etc.) of the enabled step.

For instance, a velocity value of 127 might have high color saturation (say, hot pink); a lower velocity value, say 10, would have low color saturation or even transparency, leaving only the outline of the button.

Alternatively, you could indicate velocity by filling the enabled step with color, starting from the bottom.
For example, a step with a value of 127 would be totally filled in with color; a velocity value of 64 would be filled to about half (a value of ~64).

**Comment by pierluigi** [comment]
Yep the saturation/intensity button color is exactly what I had in mind. I guess it would make a very simple interaction: click a button once to activate it to 127 velocity. Shift-click the same button to cycle through a few predetermined velocity values (low, mid, high).

**Comment by glitched01** [comment]
That would work!
Alternatively, one could click+drag (up/down) on the step to increase or decrease values.

If you didn't want it to be too granular, "high, medium, low" type of values would be fine.
